content/images/figures/convolutional_neural_network/kernel filters.svg

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
from my own past experience, i find spaces in filenames (or other characters that need to be escaped) are something to always avoid